File: String/1408. String Matching in an Array.cpp
class Solution {
public:
    vector<string> stringMatching(vector<string>& words) {
        vector< string > temp;
        vector< string > temp2;
        int i,j=0;
        for( i = 0; i<words.size(); i++ )
        {
            for( j = 0; j<words.size(); j++ )
            {
                if(i==j)
                    continue;
                if(strstr(words[j].c_str(),words[i].c_str()))
                {
                    temp.push_back(words[i]);
                    break;
                }
                    

            }
            
        }
        return temp;
        
    }
};

File: Array/1408. String Matching in an Array.py
class Solution(object):
    def stringMatching(self, words):
        """
        :type words: List[str]
        :rtype: List[str]
        """
        new = list()
        for word in words:
            for word2 in words:
                if word == word2:
                    continue
                if word in word2:
                    new.append(word)
                    break
        return new